editor-README.md

```diff
@@ -2,16 +2,16 @@ text: {
 # How to use the editor
 
 * Write in the editor
-* click the "DO" Button
```

Comment:
Commit is missing the "discussion" link, was intended to be:

  https://rebol.metaeducation.com/t/pushing-the-potential-of-polyglot-polymorphic-do/1846